guides/micronaut-webjars-stimulus/metadata.json

```diff
@@ -0,0 +1,16 @@
+{
+  "title": "Stimulus with WebJars",
+  "intro": "Learn how to load Stimulus Javascript library with WebJars",
+  "authors": ["Sergio del Amo"],
+  "categories": ["Views"],
+  "publicationDate": "2024-01-08",
+  "languages": ["java"],
+  "buildTools": ["gradle"],
```

Comment:
This works as-is with Maven, is there a reason for limiting it to gradle?

Reply:
no reason. https://github.com/micronaut-projects/micronaut-guides/pull/1412/commits/02652ec9f0e4716d65ba4fc2ae6b9676f7e308ee